test/e2e/cypress/integration/host_details.js

```diff
@@ -0,0 +1,48 @@
+import { hostName, agentId, sapSystem, clusterName, resourceGroup } from '../fixtures/host-details/selected_host'
+
+context('Host Details', () => {
+    const selectedHostName = hostName()
+    const selectedHostAgentId = agentId()
+    const selectedHostSapSystem = sapSystem()
+    const selectedHostClusterName = clusterName()
+    const selectedResourceGroup = resourceGroup()
+    
+    before(() => {
+        cy.resetDatabase()
+        cy.loadScenario('healthy-27-node-SAP-cluster')
+
+        cy.visit('/');
+        cy.navigateToItem('Hosts')
+        cy.intercept('GET', `/hosts/${selectedHostAgentId}`).as('getHostDetails')
+        cy.get(`#host-${selectedHostName} > .tn-hostname > a`).click()
+        cy.wait('@getHostDetails')
+        cy.url().should('include', `/hosts/${selectedHostAgentId}`);
+    })
+
+    describe('Detailed view for a specific host should be available', () => {
+        it('should show the host I clicked on in the overview', () => {
+            cy.get('.border-top > :nth-child(1) > .col-sm-12 > .row > :nth-child(1) > .text-muted').should('contain', 'vmhdbprd01')
```

Comment:
consider whether `vmhdbprd01`, `1154f7678ac587e5f0f242830a5201f1` or other hardcoded values here might be part of the imported `selected_host`